### PR TITLE
About section and Contact page ready for review

### PR DESCRIPTION
**Pull request by Joshsilas**
Ready for review

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -92,16 +92,42 @@ p.about{
...
+	margin-left:15px;
+}
+
+section.contact-tidy{
+    margin-top: -20px;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    align-content: stretch;
+    justify-content: space-around
```
Comment (CMeier556) [inline style.css]: Not sure if this is intentional but the content in your contact info footer is not spaced evenly.

File: index.html [hunk 2]
```diff
@@ -19,14 +19,27 @@
...
+	 					<p class="about"> Alright Mate. I am Josh. I have lived a life of organised chaos and have now found myself in the depths of coding.</p>
+						<p class="about"> Alright Mate. I am Josh. I have lived a life of organised chaos and have now found myself in the depths of coding.</p>
+						<p class="about"> Alright Mate. I am Josh. I have lived a life of organised chaos and have now found myself in the depths of coding.</p>
+						<p class="about"> Alright Mate. I am Josh. I have lived a life of organised chaos and have now found myself in the depths of coding.</p>
+						<p class="about"> Alright Mate. I am Josh. I have lived a life of organised chaos and have now found myself in the depths of coding.</p>	
 	 				<img class="about-img" src="josh.png" alt="A Picture of Josh">	
-	 		</section>
+	 			</section>
+				 </div>
+
```
Comment (CMeier556) [inline index.html:32]: Perhaps some neater indentation?

File: style.css [hunk 3]
```diff
@@ -92,16 +92,42 @@ p.about{
...
 }
 
 @media screen and (max-width: 700px) {
+	section.contact{
+   width: 100%;
+    height: 300px;
+  	background-color:#322245;
+	color: white;}
```
Comment (CMeier556) [inline style.css:125]: Maybe clean this up a bit

**Review by CMeier556 — CHANGES_REQUESTED** [review]
Not too much to do, other than a bit of cleaning and re-styling. Nice work!

File: style.css (hunk 3 above)
Comment (Joshsilas) [reply style.css:125]: Tidied and improved

File: index.html (hunk 2 above)
Comment (Joshsilas) [reply index.html:32]: yeah yeah sorted

**Review by CMeier556 — APPROVED** [review]

**Pull request merged**